app/frontend/JS/map_setup.js

```diff
@@ -0,0 +1,172 @@
+// Initialize and add the map
+let map;
+
+async function initMap() {
+  // The location of Uluru
+  const position = { lat: 40.7, lng: -74 };
+  // Request needed libraries.
+  //@ts-ignore
+  const { Map } = await google.maps.importLibrary("maps");
+  const { AdvancedMarkerView } = await google.maps.importLibrary("marker");
```

Comment:
I noticed that you are importing the "AdvancedMarkerView" from the library, but not using it in the code. If you do not plan to use this element, you can remove the line const { AdvancedMarkerView } = await google.maps.importLibrary("marker");.

Reply:
this file is deprecated, will delete it now